app/config/FrontendAppConfig.scala

```diff
@@ -48,6 +48,7 @@ class FrontendAppConfig @Inject() (override val runModeConfiguration: Configurat
   lazy val loginContinueUrl: String = loadConfig("urls.loginContinue")
   lazy val bindingTariffClassificationUrl: String = baseUrl("binding-tariff-classification")
   lazy val bindingTariffFileStoreUrl: String = baseUrl("binding-tariff-filestore")
+  lazy val pdfGeneratorUrl: String = baseUrl("pdf_generator_service")
```

Comment:
Minor but we usually go for `-` instead of `_` for these

Reply:
It will need to be added to `app-config-base` if you havent already

Reply:
Done